include/oneapi/dpl/experimental/kt/esimd_radix_sort.h

```diff
@@ -11,7 +11,7 @@
 #define _ONEDPL_KT_ESIMD_RADIX_SORT_H
 
 #include "kernel_param.h"
-#include "esimd_radix_sort_dispatchers.h"
+#include "internal/esimd_radix_sort_dispatchers.h"
```

Comment:
I think better to include headers from `internal` folder at last place.

Reply:
Internal headers are now included last.